src/org/zaproxy/zap/extension/todo/ExtensionTodoList.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2012 The ZAP development team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.todo;
+import java.util.ResourceBundle;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.ExtensionAdaptor;
+import org.parosproxy.paros.extension.ExtensionHook;
+
+public class ExtensionTodoList extends ExtensionAdaptor{
+	private TodoList todoList = null;
+    	private ResourceBundle messages = null;
+
+    	@Override
+	public String getAuthor() {
+		return "vishesh";
+	}
+
+	public ExtensionTodoList(){
+		super();
+		initialize();
+	}
+
+	public ExtensionTodoList(String name){
+		super(name);
+	}
+
+	/*
+	 * This method initializes this 
+	 */
+
+	private void initialize() {
```

Comment:
The whole method could be removed, it's just used in one place and the method does not do that much.